source/wp-content/themes/wporg-developer-2023/inc/redirects.php

```diff
@@ -71,7 +71,7 @@ public static function redirect_resources() {
 				'posts_per_page' => 1,
 			);
 		
-			$query = new WP_Query( $args );
+			$query = new \WP_Query( $args );
```

Comment:
I did test this one and it worked... is this fix required?

Reply:
This one isn't, as it's within a class that's within the default namespace.

Reply:
Reverted